gunrock/graph/dynamic_graph/dynamic_graph_base.cuh

```diff
@@ -46,14 +51,36 @@ struct DynamicGraphBase {
       typename std::conditional<REQUIRE_EDGES_VALUES, SlabHashGraphMapT,
                                 SlabHashGraphSetT>::type;
 
-  DynamicGraphT dynamicGraph;
+  mutable DynamicGraphT dynamicGraph;
```

Comment:
What's the `const` function that mutates `dynamicGraph` or why is this `mutable`?

Reply:
Fixed ec8326a.